### PR TITLE
Create new Command Line topic

### PR DESCRIPTION
**Pull request by Dusch4593**
### Description

Per internal discussions, this PR will kick off a new topic about the Command Line. This new topic includes existing content moved from old places as well as a new concept entry about Bash.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/command-line/concepts/terminal/terminal.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Command Line'
+  - 'Bash/Shell'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'paths/computer-science'
+---
+
+A **terminal**, or terminal emulator, is a program that allows users to run commands that are processed by the computer or another network device. Terminals can be completely text-based or a graphical user interface (GUI) where users interact with windows, buttons, and icons.
```
Comment (yangc95) [inline content/command-line/concepts/terminal/terminal.md:15]:
```suggestion
A **terminal**, or terminal emulator, is a program that allows users to run commands that are processed by the computer or another network device. Terminals can be completely text-based or have a graphical user interface (GUI) where users interact with windows, buttons, and icons.
```

File: content/command-line/command-line.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'paths/back-end-engineer-career-path'
+---
+
+The **command line** is a text-based interface used in most operating systems for a variety of tasks including:
+
+- Navigating and interacting with the computer's [file system structure](https://www.codecademy.com/resources/docs/general/file-system-structure)
+- Runing programs or launching applications written in a particular [tech stack](https://www.codecademy.com/resources/docs/general/tech-stack)
+- Configuring the command line environment (e.g., aliases and `PATH` variable)
```
Comment (yangc95) [inline content/command-line/command-line.md:14]:
```suggestion
- Navigating and interacting with the computer's [file system structure](https://www.codecademy.com/resources/docs/general/file-system-structure).
- Running programs or launching applications written in a particular [tech stack](https://www.codecademy.com/resources/docs/general/tech-stack),
- Configuring the command line environment (e.g., aliases and `PATH` variable).
```

File: content/command-line/concepts/bash/bash.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  - 'Linux'
+  - 'Unix'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'paths/computer-science'
+---
+
+The Bourne Again Shell, or **[Bash](https://www.gnu.org/software/bash/manual/bash.html)**, is a [Unix](https://www.codecademy.com/resources/docs/general/unix)-based shell program and language used for a multitude of purposes including system administration and software testing.
+
+Developed by Brian Fox in 1989, Bash was released as part of the GNU Project to replace the original Bourne Shell. It was also one of the first programs ported to [Linux](https://www.codecademy.com/resources/docs/open-source/linux).
```
Comment (yangc95) [inline content/command-line/concepts/bash/bash.md:19]: Concern: I notice Linux is in open-source. Should this be moved to `general/operating-systems` (General reorganization ref) instead?

**Review by yangc95 — COMMENTED** [review]
@Dusch4593 Did a first review! I mainly had some grammatical/structural suggestions. Looks good otherwise!

File: content/command-line/concepts/bash/bash.md (hunk 3 above)
Comment (Dusch4593) [reply content/command-line/concepts/bash/bash.md:19]: Yes, I agree. I can remove this link and replace once the new link for `general/operating-systems` exists. I was actually hoping to do this for macOS, Windows, and other operating system concepts in the future. 😄

**Review by yangc95 — APPROVED** [review]
LGTM

**Review by SSwiniarski — APPROVED** [review]
@Dusch4593 Looks good to me.

**Pull request merged**